ui/src/pages/GlobalHome.js

```diff
@@ -0,0 +1,293 @@
+          </p>
+        </div>
+        <CardActions>
+          <CardActionButtons>
+            <Button
+              className="pf-button-secondary"
+              raised
+              onClick={() => console.log('clicked!')}
+            >
+              {this.props.intl.formatMessage(messages.getStarted)}
+            </Button>
+          </CardActionButtons>
+        </CardActions>
+      </Card>
+    );
+  }
+
+  renderThirdPartyHelpCards() {
+    return (
+      <div className="globalhome-thirdpartywrapper">
+        <div className="endbars-headerline-wrapper" dir="ltr">
+          <span className="mdc-typography--overline endbars-headerline">
+            <FormattedMessage {...messages.youCanHelp} />
+          </span>
+        </div>
+        <Grid>
+          <Row>
+            <Cell desktopColumns={6}>{this.renderRespondersCard()}</Cell>
+            <Cell desktopColumns={6}>{this.renderDevelopersCard()}</Cell>
+          </Row>
+        </Grid>
+      </div>
+    );
+  }
+
+  renderRepoList() {
+    var cells = this.state.repos.map(repo => (
+      <Cell key={repo.repoId}>
+        <RepoCard repoInfo={repo} />
+      </Cell>
+    ));
+    return <Grid><Row>{cells}</Row></Grid>;
+  }
+
+  render() {
+    if (this.state.error) {
+      return <div>An error occurred</div>
+    }
+    if (!this.state.isLoaded) {
+      return <LoadingIndicator />;
+    }
+    return (
+      <div className="globalhome-wrapper">
+        {this.renderHeader()}
+        {this.renderRepoList()}
+        {this.renderThirdPartyHelpCards()}
+        <Footer />
+      </div>
+    );
+  }
+}
+
+/**
+ * RepoCard is a component for a card in the repo list.
+ */
+
+const repoCardMessages = defineMessages({
+  repoRecordCount: {
+    id: 'RepoCard.repoRecordCount',
+    defaultMessage: 'Tracking {recordCount} records.',
+    description: ('A message displaying how many data records we have in the '
+        + 'database.'),
+  },
+});
+
+class RepoCardImpl extends Component {
+  constructor(props) {
+    super(props);
+    this.goTo = this.goTo.bind(this);
+  }
+
+  goTo() {
+    this.props.history.push("/" + this.props.repoInfo.repoId);
+  }
+
+  render() {
+    var repocardImageClassName = (
+        "repocard-image repocard-imagedisplaycat" +
+        this.props.repoInfo.displayCat);
+    var recordCountContent = null;
+    if (this.props.repoInfo.recordCount > 0) {
+      recordCountContent = (
+        <p className="mdc-typography--body1 repocard-recordcount">
+          <FormattedMessage
+            {...repoCardMessages.repoRecordCount }
+            values={{
+              "recordCount": this.props.repoInfo.recordCount
+            }} />
+        </p>
+      );
+    }
+    return (
+      <Card className="repocard">
+        <CardPrimaryContent className="repocard-content" onClick={this.goTo}>
+          <div className={repocardImageClassName}>
+            <p className="mdc-typography--body1">
+              {this.props.repoInfo.title[0]}
```

Comment:
`title[0]` doesn't handle UTF-16 surrogate pairs correctly. Seems like `Array.from(title)[0]` should work:
https://stackoverflow.com/a/42596897/4339963